x/filetree/keeper/access.go

```diff
@@ -28,7 +28,7 @@ func HasViewingAccess(file types.Files, user string) (bool, error) {
 		return ok, nil
 	}
 
-	return true, nil
+	return false, nil
```

Comment:
You should just be able to skip the if statement all together and just return ok